### PR TITLE
Release 2.2.0

### PR DESCRIPTION
**Pull request by ramprasadn**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test_one_sample,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by rannick — APPROVED** [review]
LGTM!

File: nextflow_schema.json [hunk 1]
```diff
@@ -605,6 +631,27 @@
                     "fa_icon": "fas fa-align-center",
                     "enum": ["bwa", "bwamem2", "bwameme", "sentieon"]
                 },
+                "mt_aligner": {
+                    "type": "string",
+                    "default": "bwamem2",
+                    "description": "Specifies the alignment algorithm to use - available options are 'bwamem2' and 'sentieon'.",
+                    "fa_icon": "fas fa-align-center",
+                    "enum": ["bwa", "bwamem2", "sentieon"]
```
Comment (fellen31) [inline nextflow_schema.json:639]: available options are 'bwamem2' and 'sentieon'.

vs

"enum": ["bwa", "bwamem2", "sentieon"]

Comment (fellen31) [inline nextflow_schema.json:639]: available options are 'bwamem2' and 'sentieon'.

vs

"enum": ["bwa", "bwamem2", "sentieon"]

File: subworkflows/local/align.nf [hunk 2]
```diff
@@ -21,11 +21,12 @@ workflow ALIGN {
         ch_genome_dictionary     // channel: [mandatory] [ val(meta), path(dict) ]
         ch_mtshift_bwaindex      // channel: [mandatory] [ val(meta), path(index) ]
         ch_mtshift_bwamem2index  // channel: [mandatory] [ val(meta), path(index) ]
-        ch_mtshift_bwamemeindex  // channel: [mandatory] [ val(meta), path(index) ]
         ch_mtshift_fasta         // channel: [mandatory] [ val(meta), path(fasta) ]
         ch_mtshift_dictionary    // channel: [mandatory] [ val(meta), path(dict) ]
         ch_mtshift_fai           // channel: [mandatory] [ val(meta), path(fai) ]
+        val_mbuffer_mem          // integer: [mandatory] memory in megabytes
         val_platform             // string:  [mandatory] illumina or a different technology
+        val_sort_threads         // integer: [mandatory] memory in megabytes
```
Comment (fellen31) [inline subworkflows/local/align.nf:29]:
```suggestion
        val_sort_threads         // integer: [mandatory] number of sorting threads
```

File: subworkflows/local/annotate_genome_snvs.nf [hunk 3]
```diff
@@ -28,7 +28,8 @@ workflow ANNOTATE_GENOME_SNVS {
         analysis_type         // string: [mandatory] 'wgs' or 'wes'
         ch_cadd_header        // channel: [mandatory] [ path(txt) ]
         ch_cadd_resources     // channel: [mandatory] [ path(annotation) ]
-        ch_vcfanno_resources  // channel: [mandatory] [ path(resources) ]
+        ch_vcfanno_extra      // channel: [mandatory] [ [path(vcf),path(index)] ]
+        ch_vcfanno_resources  // channel: [mandatory] [ [path(vcf),path(index),...] ]
```
Comment (fellen31) [inline subworkflows/local/annotate_genome_snvs.nf:32]: What does `...` mean?

File: subworkflows/local/annotate_mt_snvs.nf [hunk 4]
```diff
@@ -19,7 +19,9 @@ workflow ANNOTATE_MT_SNVS {
         ch_cadd_header         // channel: [mandatory] [ path(txt) ]
         ch_cadd_resources      // channel: [mandatory] [ path(annotation) ]
         ch_genome_fasta        // channel: [mandatory] [ val(meta), path(fasta) ]
-        ch_vcfanno_resources   // channel: [mandatory] [ path(resources) ]
+        ch_vcfanno_extra       // channel: [mandatory] [ [path(vcf),path(index).....] ]
+        ch_vcfanno_lua         // channel: [mandatory] [ path(lua) ]
+        ch_vcfanno_resources   // channel: [mandatory] [ [path(vcf),path(index),.....] ]
```
Comment (fellen31) [inline subworkflows/local/annotate_mt_snvs.nf:24]: Same here

**Review by peterpru — APPROVED** [review]
LGTM!

**Review by fellen31 — COMMENTED** [review]
Nice! Just some small stuff!

File: subworkflows/local/annotate_genome_snvs.nf (hunk 3 above)
Comment (ramprasadn) [reply subworkflows/local/annotate_genome_snvs.nf:32]: That it contains more than one vcf and index 😅

Comment (fellen31) [reply subworkflows/local/annotate_genome_snvs.nf:32]: What do you think about `[ path(vcfs), path(indices) ]`? 😁

Comment (fellen31) [reply subworkflows/local/annotate_genome_snvs.nf]: It's still one list of multiple vcfs `[ meta, [ vcf1, vcf2, vcf ], [tbi1, tbi2, tbi3] ]` right?  and not `[ meta, vcf1, tbi1, vc2, tbi2, etc... ]`

Comment (ramprasadn) [reply subworkflows/local/annotate_genome_snvs.nf]: I never put much thought into it.. But how about [path(vcf1),path(index1),...,path(vcfn),path(indexn)] instead)?

Comment (ramprasadn) [reply subworkflows/local/annotate_genome_snvs.nf]: Nope.. Its just a list containing paths and indices.. They are not grouped in any way :)

Comment (fellen31) [reply subworkflows/local/annotate_genome_snvs.nf]: Maybe I misunderstood, I edited my comment above.

Comment (fellen31) [reply subworkflows/local/annotate_genome_snvs.nf]: Ah, okay. then I guess just  `...` is fine!

**Review by fellen31 — APPROVED** [review]

**Pull request merged**